### PR TITLE
Implement /v1/health endpoint status

### LINKED ISSUE #56: Determine what /v1/health's status field means
We don't have a concept of Tokend status. What does it mean? What does it display? How does that change the status code we return from this endpoint?

### PR DESCRIPTION
**Pull request by dgreene-r7**
This PR updates the health endpoint to present default token status as indicative of service health. Resolves #56.

### CODE REVIEW

File: test/http-server-v1.js [hunk 1]
```diff
@@ -4,13 +4,27 @@ require('./init');
...
+});
+
 class MockProvider {
   constructor() {}
 }
 
 class StorageServiceMock {
+  constructor() {
+    this.defaultToken = {
+      initialize: goodDefaultToken
```
Comment (fmitchell-r7) [inline test/http-server-v1.js]: Do we want to include a test for the negative case too e.g. have initialize throw and ensure we get a "service unavailable" response from the health end point?

File: test/http-server-v1.js [hunk 2]
```diff
@@ -4,13 +4,34 @@ require('./init');
...
+  status: STATUS.READY,
+  data: {
+    token: 'sometoken'
+  }
+});
+
+const badDefaultToken = () => Promise.reject(new TypeError('Invalid token.'));
+
+const badDataDefaultToken = () => Promise.resolve({
+  status: STATUS.PENDING,
```
Comment (fmitchell-r7) [inline test/http-server-v1.js]: Should this status be `READY` so the test hits to `!manager.data.token` case?

Comment (dgreene-r7) [inline test/http-server-v1.js]: Fixed in 90406f9.
